packages/discovery-react-components/src/utils/document/__tests__/documentUtils.spec.tsx

```diff
@@ -152,4 +153,27 @@ describe('findOffsetInDOM', () => {
     expect(beginOffset).toEqual(734);
     expect(endOffset).toEqual(107);
   });
+
+  it.only('finds offset when text spans multiple TextNodes', () => {
```

Comment:
doesn't this prevent other tests from running?

Reply:
D'oh! Developer error. Good catch.